claat/parser/md/parse.go

```diff
@@ -387,6 +389,7 @@ func addMetadataToCodelab(m map[string]string, c *types.Codelab) error {
 		case metaTags:
 			// Standardize the tags and append to the codelab field.
 			c.Tags = append(c.Tags, standardSplit(v)...)
+			sort.Strings(c.Tags)
```

Comment:
Same here. I thought it was done at the very end.
https://github.com/googlecodelabs/tools/blob/c0f3afc2f3a7436818b5163d1465f6c07e33e32d/claat/parser/md/parse.go#L213